rrodriguesc/prometheus-grafana-fastapi
# [Services] Add docker-compose services and configurations

This PR adds docker-compose file to integrate all api, prometheus, cAdvisor and grafana.

**Test plan:**
1. Check README.md and start all services.
2. Import all dashboards.
3. Interact with api and check if it is working properly.
